google/cloud/documentai_toolbox/wrappers/document.py

```diff
@@ -379,7 +411,10 @@ class Document:
         pages (List[Page]):
             A list of `Pages` in the `Document`.
         entities (List[Entity]):
-            A list of `Entities` in the `Document`.
+            A list of un-nested `Entities` in the `Document`.
+        chunks (List[documentai.Document.ChunkedDocument.Chunk]):
+            A list of document chunks extracted from a Layout Parser.
+        document_layout_blocks(List[documentai.Document.DocumentLayout.DocumentLayoutBlock])
```

Comment:
The `document_layout_blocks` argument is missing `:` and a description

Reply:
Thanks!